main/reference/vstorage-ref.md

```diff
@@ -21,8 +136,7 @@ The `published` and `bundles` keys are the most relevant to dapp development.
       published: 'for the chainStorage API; see below',
     }
 ```
-
-## vstorage: published.\* keys
+### vstorage: published.\* keys
```

Comment:
Demoting these from h2 to h3 means they don't show up in the page TOC. Is that on purpose?